user_management/ui/views.py

```diff
@@ -33,6 +34,7 @@ def dispatch(self, request, *args, **kwargs):
         return super(VerifyUserEmailView, self).dispatch(request, *args, **kwargs)
 
     def get(self, request, *args, **kwargs):
+        next = resolve_url(settings.LOGIN_REDIRECT_URL)
         self.activate_user()
         messages.success(request, self.success_message)
-        return redirect_to_login('/', login_url=self.url)
+        return redirect_to_login(next, login_url=self.url)
```

Comment:
This changes the default behaviour however redirecting to `/` is not always the desired behaviour.